docs/specification.md

```diff
@@ -219,12 +211,11 @@ These roles apply to the whole contract (hence "global"):
   - Can drop any proposal at any time.
   - Cannot be an implicit address, in other words it must be a contract.
   - There always must be exactly one `guardian`.
+* **delegate**
```

Comment:
This shouldn't be in this list of "global" role, it's explanation I think should be more similar to the removed one for the `operator`

Reply:
Make sense, I've updated this.